nixos/modules/installer/tools/nix-fallback-paths.nix

```diff
@@ -1,8 +1,8 @@
 {
-  x86_64-linux = "/nix/store/wsa13yx6kl6zzjlhjb8vmcqb6msbxym3-nix-2.24.11";
-  i686-linux = "/nix/store/rn1449hjxh77f723m5rc1skgv4paknic-nix-2.24.11";
-  aarch64-linux = "/nix/store/psrkach0v2s66rw30nxzpz7hgsqrvg28-nix-2.24.11";
-  riscv64-linux = "/nix/store/zc8clmxxi9zr7vx7y8v8kg1rsmhwsp42-nix-riscv64-unknown-linux-gnu-2.24.11";
-  x86_64-darwin = "/nix/store/nc81g85wlkzqck5mxdfbnxscavg2xh09-nix-2.24.11";
-  aarch64-darwin = "/nix/store/nqzxb4h7sbkk6876zx8yc88ahlgl5kkl-nix-2.24.11";
+  x86_64-linux = "/nix/store/3x497rk44p5zpr2vbf155dqw5xfjri1a-nix-2.25.3";
+  i686-linux = "/nix/store/z2k7nzwalwa69ghvrl3sd11z5gss91yn-nix-2.25.3";
+  aarch64-linux = "/nix/store/0qqzr73nid1lk9k3myxkg25vgypaspyf-nix-2.25.3";
+  riscv64-linux = "/nix/store/bshsqkqx3c0v3z8aiim01dr9n5r4yq0r-nix-riscv64-unknown-linux-gnu-2.25.3";
+  x86_64-darwin = "/nix/store/9w9p3zawqd1wx3shyhribar3gzkxkyq6-nix-2.25.3";
+  aarch64-darwin = "/nix/store/46p1z0w9ad605kky62dr53z4h24k2a5r-nix-2.25.3";
```

Comment:
2.25.4 is out: https://github.com/NixOS/nix/releases/tag/2.25.4

Reply:
Sure. This needs more work anyway to fix all failed builds before hand.